xof.go

```diff
@@ -0,0 +1,44 @@
+package kyber
+
+import (
+	"crypto/cipher"
+	"io"
+)
+
+// An XOF is an extendable output function, which is a cryptographic
+// primitive that can take arbitrary input like a hash, and then create
+// a stream of output, up to a limit determined by the size of the internal
+// state of the XOF.
+type XOF interface {
+	// Write absorbs more data into the hash's state. It panics if called
```

Comment:
is it the hash's state or the XOF's state?

Reply:
The underlying hash's state. Fixed to be hopefully clearer.